src/test/java/pakiet/PracownikTest.java

```diff
@@ -54,6 +54,31 @@ public void jestBogaczem_wyplataPonizejProgu_zwracaFalse() {
         assertFalse(actual);
     }
 
+    @Test
+    public void jestBiedakiem_wyplataPonizejProgu_zwracaTrue() {
+        //given
+        Pracownik typek = new Pracownik("Typkowski", 900);
+
+        //when
+        boolean actual = typek.jestBiedakiem();
+
+        //then
+        assertTrue(actual);
+    }
+
+    @Test
```

Comment:
Should be parametrized test with multiple values.

Reply:
test parametrized added